lib/kernel-manager.coffee

```diff
@@ -11,6 +11,7 @@ module.exports =
 class KernelManager
     constructor: ->
         @_runningKernels = {}
+        @_kernelSpecs = []
```

Comment:
I don't see why initialising `@_kernelSpecs = @parseKernelSpecSettings()` would be an improvement. Since on activation of Hydrogen `@updateKernelSpecs` is called anyway.

> One thing doesn't preclude the other, but it makes me think that `@updateKernelSpecs` could be split into two methods `@getKernelSpecsFromSettings` and `@getKernelSpecsFromJupyter`.

This is actually the case right now. The only difference are the method names:
`@updateKernelSpecs` is now called `@saveKernelSpecs`,
`@getKernelSpecsFromJupyter` is `@updateKernelSpecs` and
`@getKernelSpecsFromSettings` is `@parseKernelSpecSettings`.

We can rename those methods if you think this would make it clearer.